web/templatetags/web_extras.py

```diff
@@ -98,11 +98,11 @@ def nav_link(request, url_name, text):
     Returns:
         SafeText: HTML of navigation item
     """
-    html_classes = ["nav-link"]
+    html_classes = ["nav-link navbar-link link-secondary"]
     url = reverse(url_name)
     aria_current = ""
     if active_nav(request, url):
-        html_classes.append("active")
+        html_classes.append("active btn-secondary")
         aria_current = ' aria-current="page"'
```

Comment:
Could we convert `["nav-link navbar-link link-secondary"]` to `["nav-link", "navbar-link", "link-secondary"]`?  And the same on 105? m This is admittedly a knit-picky request.

Reply:
The separate strings means that we can add and remove classes easily.  This might not be necessary.  This was hastily written code, let me know if you'd rather do something else.